Employees/PermanentEmployeeJSON.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Employees.Entites;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Employees
+{
+    public class PermanentEmployeeJSON
+    {
+        public string path = @"..\..\..\..\Employees\Data\PermanentEmployee.json";
+        public PermanentEmployee PermanentEmployee5 = new PermanentEmployee("Alice surname", 50000, 5000, holidayAllowance: 19);
```

Comment:
Testing variable so that when the write to employee file is being called in the program class I will access this variable and it will be written to a JSON file.

Reply:
I'm referring to PermanentEmployee5 and 6, they are not used in the code?  If they were there for earlier testing, they should be taken out.  When you want to test your methods, look at using the unit tests project you already added.  Keep testing code and data separate from the code you are actually testing.  Test Driven Development would encourage you to write code this way from the bottom up i.e. never put any testing data in the code you are writing, but write the tests to test your code as you go.

Reply:
These variables were used in the program class where the methods in this class gets called but from now on I will do what you said and keep testing code completely separate.

Reply:
Or more accurately, write the tests to your problem then write the code around it for TDD, but I don't know many people that adhere to that 100% of the time :p

Reply:
> These variables were used in the program class where the methods in this class gets called but from now on I will do what you said and keep testing code completely separate.

Well we all do the occasional temporary code change to try stuff out :)  But they should never appear in a check-in certainly.  Good stuff!